icu4c/source/tools/genccode/genccode.c

```diff
@@ -84,6 +85,7 @@ static UOption options[]={
 #ifdef CAN_GENERATE_OBJECTS
 /*5*/UOPTION_DEF("object", 'o', UOPT_NO_ARG),
      UOPTION_DEF("match-arch", 'm', UOPT_REQUIRES_ARG),
+     UOPTION_DEF("skip-dll-export", 's', UOPT_NO_ARG),
```

Comment:
Also remember to add a line to the usage text fprintf.

Reply:
Done.